flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"],
```

Comment:
this is disappointing

Reply:
Actually, I think it works on others arches but I was not able to try it.
We can just remove it from here and check.